src/components/People.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { AppContext } from '../AppContext';
+
+export const People = React.memo(
+  () => {
+    const {
+      personName,
+      setPersonName,
+      appliedQuery,
+      filterPeople,
+      handleClick,
+    } = useContext(AppContext);
+
+    return (
+      <div className="dropdown is-active">
+        <div className="dropdown-trigger">
+          <input
+            type="text"
+            placeholder="Enter a part of the name"
+            className="input"
+            value={personName}
+            onChange={(event) => {
+              setPersonName(event.target.value);
+              appliedQuery(event.target.value);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)